drivers/input/input_kbd_matrix.c

```diff
@@ -264,13 +264,27 @@ static k_timepoint_t input_kbd_matrix_poll_timeout(const struct device *dev)
 	return sys_timepoint_calc(K_MSEC(cfg->poll_timeout_ms));
 }
 
+static bool input_kbd_matrix_is_unstable(const struct device *dev)
+{
+	const struct input_kbd_matrix_common_config *cfg = dev->config;
+
+	for (uint8_t c = 0; c < cfg->col_size; c++) {
```

Comment:
No c++ in the kernel :D

Reply:
took me a bit to get it really :-)

Reply:
@kartben do we have anything in our CoC about bad jokes? 👀

Reply:
That would not matter since this one was excellent :D